schematic/utils/load_utils.py

```diff
@@ -31,12 +31,12 @@ def export_json(json_doc, file_path):
 def load_default():
     """Load biolink vocabulary
     """
-    biothings_path = os.path.join(DATA_PATH, 'schema_org_schemas', 'biothings.jsonld')
+    biothings_path = os.path.join(CONFIG.DATA_PATH, 'schema_org_schemas', 'biothings.jsonld')
     return load_json(biothings_path)
 
 
 def load_schemaorg():
     """Load SchemOrg vocabulary
     """
-    schemaorg_path = os.path.join(DATA_PATH, 'schema_org_schemas', 'all_layer.jsonld')
+    schemaorg_path = os.path.join(CONFIG.DATA_PATH, 'schema_org_schemas', 'all_layer.jsonld')
     return load_json(schemaorg_path)
```

Comment:
Missing newline at the end of this file.

Reply:
We intend to remove `load_utils.py`, since it's exactly the same as `io_utils.py`.